src/helpers/language.ts

```diff
@@ -23,7 +31,15 @@ const localeToStringsMap : Record<string, LocaleStrings> = {
 	"de": germanStrings,
 	"pt": portugueseBrazilStrings,
 	"pt-pt": portuguesePortugalStrings,
-	"pl": polishStrings
+	"pl": polishStrings,
+	"fr-ca": canadianFrenchStrings,
+	"fil": filipinoStrings,
+	"ro": romanianStrings,
+	"sm": samoanStrings,
+	"sw": swahiliStrings,
+	"so": somaliStrings,
+	"tl": tagalogStrings,
+	"vi": vietnameseStrings
```

Comment:
I wonder if we might want to alphabetize these, too.  No functional benefit, but it may make it a tad easier to scan/find a language while maintaining this code in the future.